19/transaction_network_election_v2.0.yaml

```diff
@@ -0,0 +1,53 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Network Election Schema - Propose a network change
+required:
+- operation
+- version
+- asset
+- outputs
+properties:
+  operation: "VALIDATOR_ELECTION"
+  # version should greater than or equal the base transaction spec version
+  # - major number should be equal to the base transaction spec major version
+  # - minor number should be greater than or equal to base transaction spec minor version
+  version:
+    type: string
+    pattern: "^2\\.0$"
+  asset:
+    additionalProperties: false
+    properties:
+      data:
```

Comment:
`asset.data` is historically defined by users, not by the spec. What do you think about putting the spec keys into `asset`?

Reply:
That would lead to unnecessary code when separating `asset` and `metadata` from a transaction.

Reply:
Discussed offline: to preserve the semantics of `asset.data` across transactions of different type, we want to consider including these fields into the transaction object itself and storing them in a separate collection for efficient indexes and small transaction document size.